test/unit/services/test-get-caseload.js

```diff
@@ -117,4 +117,31 @@ describe('services/get-caseload', function () {
       expect(result.licenseCaseloadDetails[0]).to.eql(LICENSE_CASELOAD)
     })
   })
+
+  it('should call get-team-caseload and return caseload total summary for CUSTODY casetype for a team', function () {
+    var teamName = orgUnitConstant.TEAM.name
+    getTeamCaseload.withArgs(id).resolves(TEAM_CASELOAD)
+    return getCaseload(id, teamName).then(function (result) {
+      assert(getTeamCaseload.called)
+      expect(result.custodyTotalSummary).to.eql(9)
```

Comment:
Update data so values for each total are different, to show explicitly that the correct numbers are being added?